Makefile

```diff
@@ -125,8 +127,7 @@ lint: install-tools
 	${LINTER} run ./...
 
 test:
-	CGO_ENABLED=0 go test -coverprofile coverage.txt -failfast ./awscsm/... ./cfg/... ./cmd/... ./handlers/... ./internal/... ./logger/... ./logs/... ./metric/... ./plugins/... ./profiler/... ./tool/... ./translator/...
-
+	CGO_ENABLED=0 go test -timeout 15m -coverprofile coverage.txt -failfast ./awscsm/... ./cfg/... ./cmd/... ./handlers/... ./internal/... ./logger/... ./logs/... ./metric/... ./plugins/... ./profiler/... ./tool/... ./translator/...
```

Comment:
You take me down first lol https://github.com/aws/amazon-cloudwatch-agent/pull/701

Reply:
Can you merge that change?

Reply:
Its fine. I'm good to go with your PR

Reply:
I copied what you had since I was having an issue with the tests.